src/main/java/io/github/protonmc/proton/module/building/CompressedItemsModule.java

```diff
@@ -36,6 +40,17 @@ public void commonInit() {
         ModuleItems.register();
     }
 
+    @Override
+    public void registerResources(ResourceHandler resourceHandler) {
+        // i'm sorry
+        for (Field f : CompressedItemsModule.ModuleBlocks.class.getFields())
+            if (f.getName().toUpperCase().equals("COMPRESSED_NETHER_STAR")) {
```

Comment:
wtf?

Reply:
yes, wtf?

Reply:
why do you check for a specific item? :concern:

Reply:
please just generate the items for each block seperately, there's only 4 of them
i used reflection in the stained planks module only because i was going to refactor the module anyways